contracts/vaults/STETHVault/base/RibbonVault.sol

```diff
@@ -484,22 +488,51 @@ contract RibbonVault is
                 vaultParams.decimals
             );
 
+        IERC20 steth = IERC20(collateralToken.stETH());
+        uint256 startStethBalance = steth.balanceOf(address(this));
+        bool enoughSTETH = true;
+
+        if (withdrawYieldToken && withdrawAmount > startStethBalance) {
+            if (
+                withdrawAmount >
+                startStethBalance.add(
+                    collateralToken.getStETHByWstETH(
+                        collateralToken.balanceOf(address(this))
+                    )
```

Comment:
can consolidate into single vault lifecycle method for both completeWithdraw / withdrawInstantly + tests but now just checking logic